src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,471 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, View } from "react-native";
+import { useCallback, useContext, useState } from "react";
+import {
+  StyleSheet,
+  ScrollView,
+  View,
+  TouchableWithoutFeedback,
+  TouchableOpacity,
+  RefreshControl,
+} from "react-native";
 
-import { Button, WhiteSpace } from "@ant-design/react-native";
+import {
+  Button,
+  SearchBar,
+  Tag,
+  Modal,
+  Steps,
+  List,
+} from "@ant-design/react-native";
+import {
+  Feather,
+  MaterialIcons,
+  AntDesign,
+  Ionicons,
+} from "@expo/vector-icons";
 
 import { StyledText } from "@components/StyledText";
 import { LOCATION_DETAILS } from "@constants/routes";
-import { WaitTimeScreenProps } from "@navigators/WaitTimeStackNavigator";
+import { LocationDetailsScreenProps } from "@navigators/WaitTimeStackNavigator";
+import { ThemeContext } from "@contexts/theme";
+
+const Step = Steps.Step;
+const Item = List.Item;
+const Brief = Item.Brief;
```

Comment:
These are not really necessary and save very minimal amount of work. Personal opinion, but just writing `List.Item` everywhere is not any harder and more understandable.